### PR TITLE
microsoft-teams-integration 1.2.1

### PR DESCRIPTION
**Pull request by jamie1192**
TagCheck

### What to do if the build fails?

At the moment contributors do not have access to the CI workflow triggered by StepLib PRs. In case of a failed build, we ask for your patience. Maintainers of Bitrise Steplib will sort it out for you or inform you if any further action is needed.

### New Pull Request Checklist

*Please mark the points which you did / accept.*

- [x] __I will not move an already shared step version's tag to another commit__
- [x] I read the [Step Development Guideline](https://github.com/bitrise-io/bitrise/blob/master/_docs/step-development-guideline.md)
- [x] I have a test for my Step, which can be run with `bitrise run test` (in the step's repository)
- [x] I did run `bitrise run audit-this-step` (in the step's repository - note, if you don't have this workflow in your `bitrise.yml`, [you can copy it from the step template](https://github.com/bitrise-steplib/step-template/blob/master/bitrise.yml).)
- [x] I read and accept the [Abandoned Step policy](https://github.com/bitrise-io/bitrise-steplib#abandoned-step-policy)

### CODE REVIEW

File: steps/microsoft-teams-integration/1.2.1/step.yml [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+inputs:
+- card_title: $BITRISE_APP_TITLE
+  opts:
+    description: |
+      Primary MessageCard title text.
+
+      If no value is given, the default value will be the `BITRISE_APP_TITLE`.
+    summary: The heading text at the top of the MessageCard
+    title: Card Title
+- enable_debug: false
```
Comment (zsolt-marta-bitrise) [inline steps/microsoft-teams-integration/1.2.1/step.yml:39]: FYI, I suggest changing this to `"false"` to make sure it is parsed as a `string`, not a `bool`. Not a hard requirement now, but in the future we might require this.

**Review by zsolt-marta-bitrise — APPROVED** [review]
Thanks for fixing the inputs! I added one extra optional improvement opportunity to consider.

File: steps/microsoft-teams-integration/1.2.1/step.yml [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+- osx-10.10
+- ubuntu-16.04
+type_tags:
+- notification
+toolkit:
+  go:
+    package_name: github.com/amrfarid140/bitrise-step-microsoft-teams-integration
+deps:
+  brew:
+  - name: go
```
Comment (koral--) [inline steps/microsoft-teams-integration/1.2.1/step.yml:23]: Go should not be a dependency if go toolkit is used: https://discuss.bitrise.io/t/golang-as-a-dependency/6989

Comment (jamie1192) [reply steps/microsoft-teams-integration/1.2.1/step.yml:23]: can I add the change to this PR or do I need to submit as a new PR?

Comment (koral--) [reply steps/microsoft-teams-integration/1.2.1/step.yml:23]: According to the current guidelines, there should be only one commit for PR (except icon).
It is up to you if you prefer a force push or a new PR, usually a new PR is raised.

Comment (jamie1192) [reply steps/microsoft-teams-integration/1.2.1/step.yml:23]: no worries will close this and open a new PR with the changes, thanks @koral--

File: steps/microsoft-teams-integration/1.2.1/step.yml (hunk 1 above)
Comment (jamie1192) [reply steps/microsoft-teams-integration/1.2.1/step.yml:39]: @zsolt-marta-bitrise As mentioned in the other change suggestion by Karol below, i'm going to close this and open a new PR with the Go dependency removed.

Can I leave the bool -> string change out for this patch update and include it in my next update (2.0.0)?

Comment (zsolt-marta-bitrise) [reply steps/microsoft-teams-integration/1.2.1/step.yml:39]: Sure, that's totally fine ;)

**Pull request closed without merging**